changes to the prefab and the instruction to switch do inference part (#140)
Unity-Technologies/Unity-Robotics-Hub: tutorials/pose_estimation/Documentation/quick_demo_train.md

<img src="Gifs/2_aspect_ratio.gif"/>
</p>

## <a name="switch">Switching to Data Collection Mode</a>
The completed project is set up for inference mode by default, so we must switch it to data collection mode.

1. On the `Simulation Scenario` GameObject, check the `Fixed Length Scenario` component to enable it.

2. On the `Main Camera` GameObject, check the `Perception Camera (Script)` component to enable it.

> 

Reviewer: And also uncheck the ROS GameObject?

Author: good catch !

<img src="Gifs/2_aspect_ratio.gif"/>
</p>

## <a name="switch">Switching to Data Collection Mode</a>
The completed project is set up for inference mode by default, so we must switch it to data collection mode.

> 1. Click on the `ROSObjects` GameObject in the _**Hierarchy**_ tab and in the _**Inspector**_ view uncheck the GameObject.

Reviewer: ```suggestion
1. Uncheck the `ROSObjects` GameObject in the _**Hierarchy**_ tab to disable it.
```
